services/housetables/src/main/java/com/linkedin/openhouse/housetables/dto/mapper/UserTablesMapper.java

```diff
@@ -26,19 +26,24 @@
     uses = {UserTableVersionMapper.class})
 public interface UserTablesMapper {
 
+  String DEFAULT_STORAGE = "hdfs";
+
```

Comment:
considered keeping this constant in `:cluster:storage` but this hardcoding is short-lived until tables service is updated. Therefore keeping it as a local constant.